soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -76,6 +76,10 @@ config TACH_IT8XXX2
 	default y
 	depends on SENSOR
 
+config PECI_ITE_IT8XXX2
+	default y
+	depends on PECI
```

Comment:
Add `select PECI_INTERRUPT_DRIVEN` because the driver requires it.

Reply:
Understood.

Reply:
Added as follows:

```
config PECI_ITE_IT8XXX2
	default y
	depends on PECI
	select PECI_INTERRUPT_DRIVEN
```